ansible/roles/products/common/record-hand-data/tasks/main.yml

```diff
@@ -25,3 +25,19 @@
     job: "rsync -azP  --remove-source-files --exclude '*active*' {{ shadow_bag_storage_folder }}/ {{ remote_bag_user }}@{{ remote_bag_storage_IP }}:{{ remote_bag_storage_location }} >> /home/user/hand_cron_log 2>&1"
     cron_file: sr_sync_hand_data_to_nas
   become: true
+
+- cron:
+    name: Repair old active bags
+    user: user
+    hour: "19"
+    job: "for file in $(ls {{ shadow_bag_storage_folder }} | grep active);
+          do if [[ $(expr $(date '+%Y%m%d') - 7) > $(echo $file | sed -r 's/sr_record_all_hand_data_//g' | awk -F- '{print $1 $2 $3}') ]]; 
+          echo $file;
```

Comment:
Did you mean to put this before `then`?

Reply:
Artifact from my local test, the line 'echo $file;' will be removed.

Reply:
ok :)